evap/staff/tests/test_views.py

```diff
@@ -1923,6 +1923,71 @@ def test_evaluation_copy(self):
         self.assertEqual(copied_evaluation.contributions.count(), 4)
 
 
+class TestEvaluationExamCreation(WebTestStaffMode):
+    csrf_checks = False
+    url = reverse("staff:create_exam_evaluation")
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.manager = make_manager()
+        cls.course = baker.make(Course)
+        vote_start_datetime = datetime.datetime.now() - datetime.timedelta(days=50)
+        cls.evaluation = baker.make(Evaluation, course=cls.course, vote_start_datetime=vote_start_datetime)
+        cls.contributions = baker.make(
+            Contribution, evaluation=cls.evaluation, _fill_optional=["contributor"], _quantity=3, _bulk_create=True
+        )
+        cls.exam_date = datetime.date.today() + datetime.timedelta(days=10)
+        cls.evaluation.participants.set(baker.make(UserProfile, _quantity=3))
```

Comment:
For readability, I'd move this line below the `baker.make(Evaluation, ...)` line. To me, this is a modification of the evaluation, while the contributions stuff is (more) separate from the evaluation.